P5/Source/Specs/att.datcat.xml

```diff
@@ -6,52 +6,214 @@ See the file COPYING.txt for details
+      </fs>
+    </egXML>
+    <p>"NN" is the symbol for common noun used e.g. in the <ref
+        target="http://ucrel.lancaster.ac.uk/claws7tags.html">CLAWS-7 tagset</ref> defined by the
+      University Centre for Computer Corpus Research on Language at the University of Lancaster. The
+      very same data category used for tagging an early version of the British National Corpus, and
+      coming from the so-called <ref target="http://www.natcorp.ox.ac.uk/docs/c5spec.html">BNC Basic
+        (C5) tagset</ref>, uses the symbol "NN0" (rather than "NN"). Making these values semantically
+      interoperable / interchangeable would be extremely difficult without a human expert if they
+      were not anchored in a single point of an established reference taxonomy of morphosyntactic
+      data categories. In the case at hand, the string "<ref
+        target="http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545"
+        >http://hdl.handle.net/11459/CCR_C-1256_7ec6083c-23d4-224d-6f94-eecbe6861545</ref>" is both
+      a persistent identifier of the data category in question, as well as the address of the place where a shared definition
+      of "common noun" is provided.</p>
+    <p>While the symbols "NN", "NN0", and many others (often coming from languages other than
+      English) are implicitly members of the container category "part of speech", it is sometimes
+      useful not to rely on such an implicit relationship but rather use an explicit identifier
+      for that data category, to distinguish it from other morphosyntactic data categories, such
+      as "gender", "tense", etc. For that purpose, the above example uses the <att>datcat</att>
+      attribute to reference a definition of "part of speech". The reference taxonomy in this example is the 
+      <ref target="https://www.clarin.eu/content/clarin-concept-registry">CLARIN Concept
+        Registry</ref>.</p>
+    
+  </exemplum>
+  <exemplum xml:lang="en">
+    <p>If the feature structure markup exemplified above is to be repeated many times in a single
+      document, it is much more efficient to gather the persistent identifiers in a single place and to
+      only reference them, implicitly or directly, from feature structure markup. The following
+      example is much more concise than the one above and relies on the concepts of feature structure declaration and
+      feature value library, discussed in chapter <ptr target="#FS"/>.</p>
+    <egXML xmlns="http://www.tei-c.org/ns/Examples">
       <fs>
-        <f name="POS" dcr:datcat="http://www.isocat.org/datcat/DC-1345" fVal="#commonNoun" dcr:valueDatcat="http://www.isocat.org/datcat/DC-1256"/>
+        <f name="POS" fVal="#commonNoun"/>
+        <!-- ... -->
       </fs>
     </egXML>
+    <p>The assumption here is that the relevant feature values are collected in a place that the
+      annotation document in question has access to -- preferably, a single document per linguistic
```

Comment:
please use — instead of -- (also a bit further down)

Reply:
Oh dang, that's a search-and-replace in files. Will do.

Reply:
Hmm, the scope for this is wider than just the files I've modified -- just sayin'...

Reply:
done